code/modules/shuttle/dropship_hijack.dm

```diff
@@ -54,6 +54,9 @@
 		var/obj/structure/window/H = i
 		H.deconstruct(FALSE)
 
+	for(var/turf/closed/wall/almayer/reinforced/temphull/hull in GLOB.hijack_bustable_hull)
```

Comment:
as anything

would have been better to use a SSdcs signal here so other objects can use it too